test/fixtures/nodes_template/default.toml

```diff
@@ -12,15 +12,17 @@ filter = "info,network=trace,rpc=debug,sync=debug,relay=debug"
 color = true
 
 [network]
+path = "default/network"
 listen_addresses = ["/ip4/0.0.0.0/tcp/P2P_PORT"]
+public_addresses = []
```

Comment:
A question may not be relevant to this PR: why need `public_adresses` option in the network config?

Reply:
In a scenario, a node is behind some LB or gateway, this option allows the user to put the LB address.

Reply:
The public address list should be updated by identify protocol through remote peer connection like bitcoin. This way, we don’t need to worry about LB or gateway.

Reply:
It is different, identity protocol can find the address if peer's outbound IP is the same as LB address in this case, but it is high depends on the network setup.
Otherwise, identity can't find LB address.